README.md

```diff
@@ -11,6 +11,7 @@ Transparently use [Twitter Bootstrap 3][twbs3] with [CakePHP 3][cakephp].
 
 * CakePHP 3.x
 * Twitter Bootstrap 3.x
+* jQuery 1.9+
```

Comment:
Yes, but the instruction below explain how to add jQuery and Bootstrap to the HTML header. If you use a package manager like npm or bower. jQuery is not installed with Bootstrap. It's a peer dependency. For people who are jQuery haters. They may want to know this plugin uses it.